salt/modules/archive.py

```diff
@@ -485,6 +484,12 @@ def unzip(zip_file, dest, excludes=None, template=None, runas=None, trim_output=
         Comma-separated list of files not to unpack. Can also be passed in a
         Python list.
 
+    options
+        This options are only used when ``unzip`` binary is used. In this
+        function is ignored.
```

Comment:
This might be fine, but should we at least detect and throw a warning in this case?